src/main/java/com/nike/cerberus/command/CerberusCommand.java

```diff
@@ -19,9 +19,15 @@
 import com.beust.jcommander.Parameter;
```

Comment:
- [ ] Need to add logic for the proxy, its in the yaml / object model, but not wired up in this class were it needs to be.